src/components/Episodes/Episodes.jsx

```diff
@@ -9,7 +9,18 @@ import {
 
 export default function Episodes(){
 
-	const episodes = data.map(RENDER.episode);
+	const episodes = data.map( (obj, index) => {
+		//passing prev and next video src for video nav buttons
+		const prevTrailer = index !== 0 
+			? data[index-1].trailer
+			: {};
+		
+		const nextTrailer = index !== data.length-1 
+			? data[index+1].trailer
+			: {};
+
+		return RENDER.episode(obj, index, prevTrailer, nextTrailer);
```

Comment:
yerr, kill this whole arrow function - let's just use the render util - the point of it is preciseley to avoid doing all of this logic in the main component function.